uniffi_bindgen/src/bindings/swift/mod.rs

```diff
@@ -31,26 +31,32 @@ pub fn write_bindings(
     ci: &ComponentInterface,
     out_dir: &Path,
     try_format_code: bool,
+    is_testing: bool,
 ) -> Result<()> {
     let out_path = PathBuf::from(out_dir);
 
     let mut module_map_file = out_path.clone();
-    module_map_file.push(format!("uniffi_{}.modulemap", ci.namespace()));
+    module_map_file.push(config.modulemap_filename());
 
-    let mut header_file = out_path.clone();
-    header_file.push(format!("{}-Bridging-Header.h", ci.namespace()));
-
-    let mut source_file = out_path;
+    let mut source_file = out_path.clone();
     source_file.push(format!("{}.swift", ci.namespace()));
 
-    let Bindings { header, library } = generate_bindings(config, &ci)?;
+    let Bindings { header, library } = generate_bindings(config, &ci, is_testing)?;
+
+    let header_filename = config.header_filename();
+    let mut header_file = out_path;
+    header_file.push(&header_filename);
 
     let mut h = File::create(&header_file).context("Failed to create .h file for bindings")?;
     write!(h, "{}", header)?;
 
     let mut m =
         File::create(&module_map_file).context("Failed to create .modulemap file for bindings")?;
-    write!(m, "{}", generate_module_map(&ci, &header_file)?)?;
+    write!(
+        m,
+        "{}",
+        generate_module_map(config, &ci, &Path::new(&header_filename))?
```

Comment:
I wonder if we need the modulemap also? I don't think we do, so maybe we could in a future patch guard it with `is_testing`?